src/sentry/static/sentry/app/components/group/suggestedOwnerHovercard.jsx

```diff
@@ -0,0 +1,200 @@
+                </ReasonItem>
+              ))}
+            </ReasonGroup>
+          </React.Fragment>
+        )}
+        {rules !== undefined && (
+          <React.Fragment>
+            <div className="divider">
+              <h6>{t('Ownership Rules')}</h6>
+            </div>
+            <ReasonGroup>
+              {rules.map(([type, matched], i) => (
+                <ReasonItem key={i}>
+                  <OwnershipTag tagType={type} />
+                  <OwnershipValue>{matched}</OwnershipValue>
+                </ReasonItem>
+              ))}
+            </ReasonGroup>
+          </React.Fragment>
+        )}
+      </HovercardBody>
+    }
+    {...props}
+  />
+);
+
+SuggestedOwnerHovercard.propTypes = {
+  /**
+   * The suggested actor.
+   */
+  actor: PropTypes.oneOfType([
+    // eslint-disable-next-line react/no-typos
+    SentryTypes.User,
+    // Sentry user which has *not* been expanded
+    PropTypes.shape({
+      email: PropTypes.string.isRequired,
+      id: PropTypes.string.isRequired,
+      name: PropTypes.string.isRequired,
+    }),
+    // Unidentifier user (from commits)
+    PropTypes.shape({
+      email: PropTypes.string.isRequired,
+      name: PropTypes.string.isRequired,
+    }),
+    // Sentry team
+    PropTypes.shape({
+      id: PropTypes.string.isRequired,
+      name: PropTypes.string.isRequired,
+    }),
+  ]),
+
+  /**
+   * The list of commits the actor is suggested for. May be left blank if the
+   * actor is not suggested for commits.
+   */
+  commits: PropTypes.arrayOf(
+    PropTypes.shape({
+      message: PropTypes.string.isRequired,
+      dateCreated: PropTypes.string.isRequired,
+    })
+  ),
+
+  /**
+   * The list of ownership rules the actor is suggested for. Maybe left blank
+   * if the actor is not suggested based on ownership rules.
+   */
+  rules: PropTypes.arrayOf(PropTypes.array),
+};
+
+const tagColors = {
+  url: 'greenLight',
+  path: 'blueLight',
+};
+
+const CommitIcon = styled(p => <InlineSvg src="icon-commit" size="16px" {...p} />)`
+  margin-right: 4px;
+  flex-shrink: 0;
+`;
+
+const CommitMessage = styled(({message, date, ...props}) => (
+  <div {...props}>
+    {message.split('\n')[0]}
+    <CommitDate date={date} />
+  </div>
+))`
+  color: ${p => p.theme.gray5};
+  font-size: 11px;
+  margin-top: 2px;
+`;
+
+const CommitDate = styled(({date, ...props}) => (
+  <div {...props}>{moment(date).fromNow()}</div>
+))`
+  margin-top: 4px;
+  color: ${p => p.theme.gray2};
+`;
+
+const ReasonGroup = styled('ul')`
+  list-style: none;
+  margin: 0;
+  padding: 0;
+`;
+
+const ReasonItem = styled('li')`
+  display: flex;
+  align-items: flex-start;
+
+  &:not(:last-child) {
+    margin-bottom: ${space(1)};
+  }
+`;
+
+const OwnershipTag = styled(({tagType, ...props}) => <div {...props}>{tagType}</div>)`
+  background: ${p => p.theme[tagColors[p.tagType]]};
+  color: #fff;
+  font-size: 11px;
+  padding: 2px 4px;
+  margin: 2px 4px 2px 0;
+  border-radius: 2px;
```

Comment:
we have a standard borderRadius of `4px` which we like to use unless it looks bad.